spec/jobs/document_uploads/pdf_conversion_job_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe DocumentUploads::PdfConversionJob do
+  describe '#perform' do
+    before(:all) do
+      @file_path = Rails.root.join('spec/fixtures/files/pdf_conversion.jpg')
+      @file_path_converted = Rails.root.join('spec/fixtures/files/pdf_conversion.jpg.pdf')
+      MiniMagick::Tool::Convert.new do |convert|
```

Comment:
This is creating the .jpg right?  Can this be abstracted into a helper for readability?

[SO Post](http://stackoverflow.com/a/8127728)